Makefile

```diff
@@ -2,6 +2,11 @@
 USER_ID := $(shell id -u)
 GROUP_ID := $(shell id -g)
 IN_DOCKER_IMG := $(shell test -f /.dockerenv && echo 1 || echo 0)
+SECRETS := $(CURDIR)/.secrets
+export MODELON_IMPACT_CLIENT_API_KEY ?= $(shell cat $(SECRETS)/api.key)
+export JUPYTERHUB_API_TOKEN ?= $(shell cat $(SECRETS)/jupyterhub-api.key)
+export MODELON_IMPACT_CLIENT_URL ?= https://jhmi-staging.modelon.com/
+export MODELON_IMPACT_USERNAME ?= $(shell git config user.email)
```

Comment:
Please add `export MODELON_IMPACT_CLIENT_INTERACTIVE:=false`  since accidental "true" that I had overrides user's api.key file.

`export MODELON_IMPACT_USERNAME ?= $(shell git config user.email | tr '[:upper:]' '[:lower:]')`
I have apparently capital letters returned from `git config user.email` not matching lowercase in username which prevented cleaning from e-mail from recording. We should add a test that username was correctly set at some point... We could also make replacement not case sensitive.

Reply:
Good catch. I made the replacement case insensitive. :)